storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst

```diff
@@ -8,7 +8,17 @@
 
    This feature implementation is considered **ALPHA** quality.
 
-|xbcloud| is a new tool which is part of the |Percona XtraBackup| 2.3.0-alpha1 release. The purpose of |xbcloud| is to download and upload full or part of |xbstream| archive from/to cloud. Archive uploading will employ multipart upload for Amazon and Large Objects on Swift. Along with |xbstream| archive index file will be uploaded which contains list of files and their parts and offsets of those parts in the |xbstream| archive. This index is needed for downloading only part of archive (one or several tables from backups) on demand.
+|xbcloud| is a new tool which is part of the |Percona XtraBackup| 2.3.0-alpha1 release. The purpose of |xbcloud| is to download and upload full or part of |xbstream| archive from/to cloud. Archive uploading will employ multipart upload for Amazon and Large Objects on Swift. Along with |xbstream| archive index file will be uploaded which contains list of files and their parts and offsets of those parts in the |xbstream| archive. This index is needed for downloading only part of archive (one or several tables from backups) on demand. |xbcloud| will refuse to overwrite the backup with the same name.
```

Comment:
SLO are no longer used. The way backup stoked now is described in next paragraph, so just remove the sentences "Archive uploading will employ multipart upload for Amazon and Large Objects on Swift. Along with |xbstream| archive index file will be uploaded which contains list of files and their parts and offsets of those parts in the |xbstream| archive. This index is needed for downloading only part of archive (one or several tables from backups) on demand."

In order to download backup we use LIST API call to get the list of chunks.